types/relay-runtime/index.d.ts

```diff
@@ -728,7 +734,7 @@ interface ReaderCondition {
 }
 
 interface ReaderClientExtension {
-    readonly kind: 'ClientExtension';
+    readonly kind: string; // 'ClientExtension';
```

Comment:
Why was this change necessary?

Reply:
> Why was this change necessary?

Just to match all the other members in the union `ReaderSelection`. I would prefer to have a more strict type than just `string`, but this means that others should be refactored, and possible some other members added to the union that now match `string`.

Reply:
Gotcha :+1: